### PR TITLE
Divider Component updated to v1 API

### PR DESCRIPTION
**Pull request by jhchill666**
- Divider Component updated to v1 API
- Documents updated to reflect
- Divider-test added to suite

### CODE REVIEW

File: src/elements/Divider/Divider.js [hunk 1]
```diff
@@ -1,30 +1,81 @@
...
+  const _children = createFragment({
+    children,
+  })
+
+  const DividerComponent = 'div'
+  const rest = getUnhandledProps(Divider, props)
+
+  return (
+    <DividerComponent className={classes} {...rest}>
+      {_children}
```
Comment (levithomason) [inline src/elements/Divider/Divider.js]: Can we use the children directly here? Not sure why the fragment is required.

**Comment by levithomason** [comment]
Wow, nice work! Thanks. Just the one question about the fragment then we can merge this guy.

File: test/specs/elements/Divider/Divider-test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import Divider from 'src/elements/Divider/Divider'
+import * as common from 'test/specs/commonTests'
+
+describe('Divider', () => {
+  common.isConformant(Divider)
```
Comment (levithomason) [inline test/specs/elements/Divider/Divider-test.js:7]: Let's add common.rendersChildren and common.hasUIClassName.

Comment (levithomason) [inline test/specs/elements/Divider/Divider-test.js:7]: There are also common tests for the classNames, reference https://github.com/TechnologyAdvice/stardust/blob/master/test/specs/elements/Label/Label-test.js

File: src/elements/Divider/Divider.js (hunk 1 above)
Comment (jhchill666) [inline src/elements/Divider/Divider.js]: Of course.  I'd just whittled down the Label version, and not considered `createFragment` was now redundant

**Comment by jhchill666** [comment]
Ok there you have it @levithomason

**Comment by levithomason** [comment]
👍

**Comment by levithomason** [comment]
I've added #269 to track the remaining components needing updated.  I've also added a section to the README outlining ["How can I help?"](https://github.com/TechnologyAdvice/stardust#how-can-i-help).

Your PR was much appreciated, would love to see more!

**Pull request merged**